daal4py/sklearn/linear_model/logistic_path.py

```diff
@@ -381,8 +445,11 @@ def __logistic_regression_path(
                     daal_extra_args_func = _daal4py_cross_entropy_loss_extra_args
                 func = _daal4py_loss_and_grad
             else:
-                def func(x, *args):
-                    return _multinomial_loss_grad(x, *args)[0:2]
+                if sklearn_check_version('1.1'):
```

Comment:
```suggestion
                if sklearn_check_version('1.1') and loss is not None:
```

Reply:
I think explicitly specifying `loss = None` is safer